css/src/components/form/radio.scss

```diff
@@ -10,62 +9,35 @@ $radio-duration: $input-transition-duration !default;
 $radio-animation: boop !default;
 $radio-spacing: 0.5em !default;
 
-@mixin radio-dot {
-	position: absolute;
-	inset: 0;
-	width: $radio-dot-size;
-	height: $radio-dot-size;
-	margin: auto;
-	border-radius: $border-radius-rounded;
-	background-color: $body-background;
-	content: '';
-	overflow: hidden;
-}
-
 @mixin radio-dot-checked {
 	background: $radio-dot-checked-color;
+	box-shadow: inset 0 0 0 0.25em $body-background;
```

Comment:
The dots disappear in forced colors mode

Reply:
might be able to use this technique https://github.com/microsoft/atlas-design/blob/main/css/src/components/toggle.scss#L25

Reply:
yup looking at that right now

Reply:
Forced colors sets box shadow to none, so I think we can use the previous strategy of `:before` for forced colors only. This means the dot can still look weird in forced colors mode though. 

![image](https://user-images.githubusercontent.com/94572161/162494388-e8d50991-a736-49ff-bd4c-ebeda0b941d2.png)

Reply:
That will work, just verifying `forced-color-adjust: none;` was used inside the mixin body during testing though before we go that route.

```scss
@include forced-colors {
  forced-color-adjust: none;
  [... etc]
}
```

Reply:
Tested using both with and without `forced-color-adjust: none;` using all four OS high contrast settings.